daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/NodeInfo.scala

```diff
@@ -517,15 +551,15 @@ object NodeInfo extends Enum {
         }
         f
       }
-      override def fromNumber(n: Number): DataValueFloat = n.floatValue
+      override def fromNumberNoCheck(n: Number): DataValueFloat = n.floatValue
```

Comment:
Hmmm. Scala question - If you override a protected member, but don't say protected, did you just widen things to public, or does it inherit the protectedness?

Reply:
Just did a test. Not the behavior I expected, but the answer is it does not inherit the protectedness. It does in fact widen the visiblity.

So these overides shoudl also include ``protected``.